compiler/lib/src/main/scala/codegen/DictionaryJsonWriter/DictionaryJsonEncoder.scala

```diff
@@ -13,10 +13,10 @@ import io.circe.syntax._
  *  ====================================================================== */
 case class DictionaryMetadata(
     deploymentName: String = "",
-    projectVersion: String, 
-    frameworkVersion: String, 
-    libraryVersions: List[String], 
-    dictionarySpecVersion: String
+    projectVersion: String = "", 
```

Comment:
Instead of "", can we make this something like "[ no value specified ]"?

Reply:
Added the recommendation